vault/request_handling.go

```diff
@@ -1992,3 +1993,34 @@ func (c *Core) checkSSCTokenInternal(ctx context.Context, token string, isPerfSt
 	// status code.
 	return "", logical.ErrMissingRequiredState
 }
+
+func (c *Core) GetUserLockoutFromConfig(logicalType string) UserLockoutConfig {
```

Comment:
Where is this function used?

Reply:
I added this function so that it could be used in an upcoming Jira ticket. I can remove this function from this PR.